osf/models/analytics.py

```diff
@@ -158,9 +158,24 @@ def update_counter(cls, page, node_info):
             model_instance.save()
 
     @classmethod
-    def get_basic_counters(cls, page):
-        try:
-            counter = cls.objects.get(_id=cls.clean_page(page))
-            return (counter.unique, counter.total)
-        except cls.DoesNotExist:
-            return (None, None)
+    def _get_file_counter(cls, counter_type, file_id, version=None):
+        """
+        Pulls the total file count for views or downloads across different nodes.
+        :param counter_type: String
+        :param file_id: String
+        :param version: Int the version number
+        :return: Int the total count
+        """
+        if version is not None:
+            file_id += ':' + str(version)
```

Comment:
It's generally a bad idea to "mutate" function arguments if you can avoid it (strings are immutable so you're not actually mutating file_id, but the concept still holds). The caller would not expect a "getter" function to have side effects.

Also, use `str.format`, as per style guide: https://cosdev.readthedocs.io/en/latest/style_guides/python.html#string-formatting .